# Point Of Interest message

# ID of the point of interest
int32 point_id

# ID of the region associated with the point of interest
int32 region_id

# Coordinates of the point of interest
Coordinate coordinates

# Task associated with the point of interest
string task

# Status of the point of interest
string status
